OpenRA.Mods.Common/Projectiles/Missile.cs

```diff
@@ -159,6 +147,39 @@ public class MissileInfo : IProjectileInfo
 			"not trigger fast enough, causing the missile to fly past the target.")]
 		public readonly WDist CloseEnough = new WDist(298);
 
+		[Desc("Length of the contrail (in ticks).")]
+		public readonly int ContrailLength = 0;
+
+		[Desc("Offset for contrail's Z sorting.")]
+		public readonly int ContrailZOffset = 2047;
+
+		[Desc("Delay of the contrail.")]
+		public readonly int ContrailDelay = 1;
+
+		[Desc("Width of the contrail.")]
+		public readonly WDist ContrailWidth = new WDist(64);
+
+		[Desc("RGB color when the contrail starts.")]
+		public readonly Color ContrailStartColor = Color.White;
+
+		[Desc("Use player remap color instead of a custom color when the contrail starts.")]
+		public readonly bool ContrailStartColorUsePlayerColor = false;
+
+		[Desc("The Alpha value of color when the contrail starts.")]
+		public readonly int ContrailStartColorAlpha = 255;
+
+		[Desc("RGB color when the contrail ends.")]
+		public readonly Color ContrailEndColor = Color.White;
+
+		[Desc("Use player remap color instead of a custom color when the contrail ends.")]
+		public readonly bool ContrailEndColorUsePlayerColor = false;
+
+		[Desc("The Alpha value of color when the contrail ends.")]
```

Comment:
```suggestion
		[Desc("The Alpha value [from 0 to 255] of color when the contrail ends.")]
```

I recon it would be useful to mention this next to every alpha value as we are integers instead of hexcode

Reply:
sure, will fix it 8 hours later

Reply:
fixed